packages/boxel-ui/addon/src/components/switch/index.gts

```diff
@@ -67,7 +67,7 @@ export default class Switch extends Component<SwitchSiganture> {
         );
       }
       .switch.checked .switch-input {
-        transform: translateX(10px);
+        margin-left: 49%;
```

Comment:
I had a secondary goal of making the smaller element entirely sized relative to the larger one vs using specific numbers, but this was _slightly_ off with 50% 😬